app/jobs/alert_email/base.rb

```diff
@@ -1,19 +1,20 @@
 class AlertEmail::Base < ApplicationJob
+  MAXIMUM_RESULTS_PER_RUN = 500
```

Comment:
Is this still needed? I only see [this removed call](https://github.com/DFE-Digital/teaching-vacancies/pull/7318/files#diff-bad11ea43382a817d4981b22e8afb2e5e790dd4c339d39866443649920bd107eL46)

Reply:
It was - I've out the usage back.